xml/s4s_access.xml

```diff
@@ -125,6 +117,17 @@
 
   <procedure>
    <title>Using &hana-fw;</title>
+   <remark>toms 2019-05-28: This needs to be reworked. According
+   to Sören:
+   * There is no proposal step anymore [...]
+   * Allowing Remote Shell Access (SSH) is not part of HANA Firewall anymore.
+   * It is not possible to choose network interfaces anymore.
+     Also it is not called "Allowed Services".
+   * Adding services have to be done via command line (hana-firewall
+     define-new-hana-service).
+   * There is no integration in YaST.
+   * The screenshot is wrong.
+   </remark>
```

Comment:
Fwiw, I only ever faked my HANA firewall setups by manipulating some checks in the Ruby code of the HANA-FW YaST module :/ iow, I would be no use whatsoever in helping to actually set up this tool.